Resources/views/Tree/tree.html.twig

```diff
@@ -24,15 +24,23 @@ file that was distributed with this source code.
 <script type="text/javascript">
     jQuery(function($) {
         $('#tree').cmfTree({
+            dnd: { enabled: true },
             request: {
                 load: function (nodePath) {
                     return {
-                        url: '{{ path('_cmf_resource', {
+                        url: '{{ path('_get_cmf_resource', {
                             repositoryName: repository_name,
                             root_node: root_node,
```

Comment:
is the github parser not capable of handling multiline strings? oh, or is this actually wrong and would need quotes? i think this assumes its twig variables, no?